### PR TITLE
Updates library.json

### PR DESCRIPTION
**Pull request by mightea**
This PR adds the following devices:
 - eWeLink SNZB-02P Temperature and Humidity Sensor
 - frient A/S SMSZB-120 Smoke Alarm
 - ZHA model names for[Signify Netherlands B.V. devices

### CODE REVIEW

File: custom_components/battery_notes/data/library.json [hunk 1]
```diff
@@ -271,6 +276,11 @@
             "battery_type": "AA",
             "battery_quantity": 4
         },
+        {
+            "manufacturer": "by frient A/S",
```
Comment (andrew-codechimp) [inline custom_components/battery_notes/data/library.json:280]: Is the leading "by " definitely there ?

Comment (mightea) [reply custom_components/battery_notes/data/library.json:280]: At least here:
image

Comment (mightea) [reply custom_components/battery_notes/data/library.json:280]: Ah sorry, my bad... one sec...

Comment (andrew-codechimp) [reply custom_components/battery_notes/data/library.json]: Thanks, easy mistake to make copy/pasting.

**Pull request merged**